### PR TITLE
WIP - DO NOT MERGE - Add ConductR specifics

### PR DESCRIPTION
**Pull request by huntc**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -67,9 +69,16 @@ lazy val frontEnd = project("front-end")
...
       "org.webjars" % "react-router" % "1.0.3",
       "org.webjars" % "jquery" % "2.2.0",
-      "org.webjars" % "foundation" % "5.3.0"
+      "org.webjars" % "foundation" % "5.3.0",
+      "com.typesafe.conductr" %% "lagom10-conductr-bundle-lib" % "1.4.0"
     ),
-    ReactJsKeys.sourceMapInline := true
+    ReactJsKeys.sourceMapInline := true,
+    // ConductR settings
+    BundleKeys.nrOfCpus := 1.0,
```
Comment (markusjura) [inline build.sbt:77]: What about changing `sbt-lagom-bundle` such that it sets these settings not only for `LagomJava` but also for `LagomPlay`?

Comment (huntc) [inline build.sbt:77]: I think that the correct place to do that would be in an `sbt-play-bundle` plugin - quite agnostic to Lagom.

Comment (huntc) [inline build.sbt:77]: Perhaps create an issue for it?

Comment (markusjura) [inline build.sbt:77]: `LagomPlay` is part of the `LagomPlugin`. Therefore it should be fine to add that to `sbt-lagom-bundle`.

Comment (markusjura) [inline build.sbt:77]: Created an issue for that: https://github.com/typesafehub/sbt-lagom-bundle/issues/14

**Comment by markusjura** [comment]
Can this PR be closed? The PR https://github.com/lagom/activator-lagom-java-chirper/pull/3 merged it to https://github.com/lagom/activator-lagom-java-chirper directly.

**Pull request closed without merging**